### DISCUSSION TITLE
How to isolate datasets in PCA plot?

### DISCUSSION THREAD

**Discussion by Mrfa0811** [Nesvilab/FragPipe-Analyst-doc#32] [Q&A] [answered]
Hi,

I've done DIA analysis on my proteomics dataset, and am interpreting the results. Fragpipe generates a PCA plot with a color-coded legend for each group. However, I have 10 total groups being compared and so having all 10 show on the plot is messy. I am trying to individually select the groups I would like to show up on the plot. The tooltip says that double-clicking on the legend allows for individual selection of datasets, but as far as I can tell this does not work. I am only able to show or remove every group at once. Is there a way to only show specific groups on this plot? Thanks!

**Comment by hsiaoyi0504** [comment, marked as answer]
Thanks for mentioning that. I will need to look into it. It looks like plotly doesn't act correctly.
